src/solve.jl

```diff
@@ -28,7 +28,7 @@ function init{uType,tType,isinplace,algType<:Union{AbstractRODEAlgorithm,Abstrac
               qoldinit=1//10^4, fullnormalize=true,
               beta2=beta2_default(alg),
               beta1=beta1_default(alg,beta2),
-              delta=1//6,maxiters = 1e9,
+              delta=map(x->1//6,reltol),maxiters = 1e9,
```

Comment:
Because of the type definition of SDEOptions in https://github.com/JuliaDiffEq/StochasticDiffEq.jl/blob/master/src/options_type.jl: `delta` and `reltol` must be of the same type. Without this change `delta` has also to be specified explicitly as a vector if `reltol` is a vector.

But probably a better solution to that problem would be changing the types of `abstol`, `reltol`, and `delta` in https://github.com/JuliaDiffEq/StochasticDiffEq.jl/blob/master/src/options_type.jl to e.g. `A`, `R`, and `D`, respectively.